### PR TITLE
FINERACT-1707: Transaction Summary Report changes

### PR DESCRIPTION
**Pull request by josehernandezfintecheandomx**
## Description

Transaction Summary report updated with the following changes

- To have Transaction subtype column ( this is payment types in fineract)
- To have Accural transaction split based on the charge types)
- Payment type_name = which is nothing but the transaction subtype of the repayments, chargebacks, and any transaction type which is using the payment type dropdown.
- Charge type = This displays the split of accrual transactions between the types of charges

[FINERACT-1707](https://issues.apache.org/jira/browse/FINERACT-1707)

Screenshot 2023-02-16 at 23 19 07

## Checklist

Please make sure these boxes are checked before submitting your pull request - thanks!

- [ ] Write the commit message as per https://github.com/apache/fineract/#pull-requests

- [ ] Acknowledge that we will not review PRs that are not passing the build _("green")_ - it is your responsibility to get a proposed PR to pass the build, not primarily the project's maintainers.

- [ ] Create/update unit or integration tests for verifying the changes made.

- [ ] Follow coding conventions at https://cwiki.apache.org/confluence/display/FINERACT/Coding+Conventions.

- [ ] Add required Swagger annotation and update API documentation at fineract-provider/src/main/resources/static/legacy-docs/apiLive.htm with details of any API changes

- [ ] Submission is not a "code dump".  (Large changes can be made "in repository" via a branch.  Ask on the developer mailing list for guidance, if required.)

FYI our guidelines for code reviews are at https://cwiki.apache.org/confluence/display/FINERACT/Code+Review+Guide.

### CODE REVIEW

**Comment by bharathcgowda** [comment]
@adamsaghy @alexivanov could any one of you please help in reviewing this PR?

File: fineract-provider/src/main/resources/db/changelog/tenant/parts/0092_update_general_accounting_table_reports.xml [hunk 1]
```diff
@@ -0,0 +1,195 @@
...
+    under the License.
+
+-->
+<databaseChangeLog xmlns="http://www.liquibase.org/xml/ns/dbchangelog"
+    xmlns:xsi="http://www.w3.org/2001/XMLSchema-instance"
+    xsi:schemaLocation="http://www.liquibase.org/xml/ns/dbchangelog http://www.liquibase.org/xml/ns/dbchangelog/dbchangelog-4.3.xsd">
+    <!-- Transaction Summary report -->
+    <changeSet author="fineract" id="3">
+        <update tableName="stretchy_report">
+            <column name="report_sql" value="SELECT
```
Comment (adamsaghy) [inline fineract-provider/src/main/resources/db/changelog/tenant/parts/0092_update_general_accounting_table_reports.xml]: is this working with mysql and postgres as well?

**Comment by adamsaghy** [comment]
@josehernandezfintecheandomx Kindly fix the conflict and please help me check whether this report is working with mysql and postgres as well? (should we have a test which calls it and check some outcome maybe?)

**Comment by bharathcgowda** [comment]
@adamsaghy  could you please review again and merge please?

**Comment by adamsaghy** [comment]
> @adamsaghy could you please review again and merge please?

I am still waiting on @josehernandezfintecheandomx reply on my questions...

**Comment by bharathcgowda** [comment]
@adamsaghy  the script works both on mysql and postgres.

**Pull request merged**